chromium-armhf

```diff
@@ -6,6 +6,14 @@ addrip="$(docker network inspect bridge --format='{{(index .IPAM.Config 0).Gatew
 echo "Enabling XHost Forwarding"
 xhost +local:docker
 
+if pacmd list-modules|grep native-protocol-tcp >/dev/null;then
+  echo Pulseaudio native-protocol-tcp Checked
+else
+  echo Loading Pulseaudio native-protocol-tcp module
+  pacmd load-module module-native-protocol-tcp
+  echo Pulseaudio native-protocol-tcp Loaded
+fi
+
```

Comment:
This little snippet of code will do (almost) the same plus will handle a possible failure:
```
echo -n "Pulseaudio native-protocol-tcp: "
if pacmd list-modules|grep native-protocol-tcp >/dev/null; then
  echo Checked
else
  pacmd load-module module-native-protocol-tcp && echo Loaded || echo Failed
fi
```

Reply:
Yes, more checking is always better.
However, I also found out that "pacmd load-module" seems to always return zero...

Reply:
My bad, but pactl would return non-zero exit codes, if it fails. With patctl this will look:
```
echo -n "Pulseaudio native-protocol-tcp: "
if pactl list modules|grep native-protocol-tcp >/dev/null; then
  echo Checked
else
  pactl load-module module-native-protocol-tcp >/dev/null && echo Loaded || echo Failed
fi
```
pactl is included in libpulse package, which is installed with pulseaudio package because of dependency, so it will be there.

Reply:
updated.